src/less/three-d-secure.less

```diff
@@ -35,8 +35,6 @@
     border-radius: 3px;
     margin: auto;
     height: 450px;
-    width: 400px;
```

Comment:
why are these getting removed?

Reply:
I'm curious about that too.

Reply:
See lines 33-34 of this file (duplicates)

Reply:
:+1:

Reply:
Ahhhhhhh thanks, GH cut that part off